.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,66 +1,32 @@
-<!--
-  ☝️How to write a good PR title:
-  - Prefix it with [ComponentName] (if applicable), for example: [Button]
-  - Start with a verb, for example: Add, Delete, Improve, Fix…
-  - Give as much context as necessary and as little as possible
-  - Prefix it with [WIP] while it’s a work in progress
--->
+<!-- PR title: [ComponentName] *Verb* a ... -->
 
 ### WHY are these changes introduced?
 
-Fixes #0000 <!-- link to issue if one exists -->
-
-<!--
-  Context about the problem that’s being addressed.
--->
+Fixes #0000
 
 ### WHAT is this pull request doing?
 
-<!--
-  Summary of the changes committed.
+Before / after screenshots are appreciated for UI changes.
+Include alt text that describes the screenshot.
+Put any gifs inside a `<details>` block.
```

Comment:
```suggestion
Put any gifs inside a `<details>` block with a `<summary>` to create a toggle.
```

Adding this specifically so the content can be hidden with the automatically created toggle. With just a `<details>` element, that won't happen.